stl/debugger/STL.natvis

```diff
@@ -1576,6 +1575,53 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <Type Name="std::timed_mutex">
+    <AlternativeType Name="std::recursive_timed_mutex"/>
+    <DisplayString>{_My_mutex}</DisplayString>

```

Comment:
This is incorrect - `_My_mutex` is an internal detail of the lock functions, and doesn't have anything to do with whether the mutex is locked or not. `_My_locked` is how we know whether the mutex is locked.